Game_Manual/Echo.md

```diff
@@ -18,4 +18,47 @@ For many commands, passing the `-h` or `--help` flag will show you a helpful mes
 
 For example, to get help with the `/review` command, type `/review --help`.
 
-[los-repo]: https://github.com/LearnersGuild/los
+It's also the primary means by which we communicate with each other - we don't send email at Learners Guild. Because we have so many people communicating on a public platform, it's important to observe some shared Guidelines:
+
+## Guidelines
+
+These guidelines are to support us in making the most of this tool while minimizing distractions and maximizing learning:
+
+Managing **OUTPUT**: guidance for managing the volume of posts & notifications we generate:
```

Comment:
This feels like a section heading to me, with a descriptive blurb?
```md
### Managing Output
This section provides guidance for managing the volume of posts and notifications we generate.
```
I also replaced the & with and because my [English teachers were mean to me](http://www.betterwritingskills.com/tip-w002.html).